### ISSUE TITLE
It is not possible to render a standalone README.rst / README.md with this Docker image

### ISSUE THREAD

**Issue by sypets** [t3docs/docker-render-documentation#31]
output:

```ATTENTION:

   No documentation found! No documentation rendered!

   Reason: None of the possible starting files (called 
   "masterdoc") could not be found. Please provide at
   least one of the following. They will be taken into
   account in this order of preference:

      1. Documentation/Index.rst
      2. Documentation/index.rst
      3. Documentation/Index.md
      4. Documentation/index.md
      5. README.rst
      6. README.md
      7. doc/manual.sxw

```

But README.rst (see number 5) does exist! So actually, according to the output it should work, right?

What did work was:
1. Documentation/Index.rst
2. Documentation/index.rst

Documentation currently also lists standalone readme as option: https://docs.typo3.org/typo3cms/HowToDocument/Appendix/SupportedFilenamesAndFormats.html

If this should no longer be supported, we should check how many extensions this affects and update the documentation.

**Comment by sypets** [comment]
@marble @DanielSiepmann Can someone with write access to this repo please add the label "bug" to this issue?

Also, I don't know how many extensions this affects, but can you please consider adding a label "high priority" or "severe" or something else and add it to this issue? 

I will now close some of the more trivial nice-to-have doc issues (from myself).

**Comment by marble** [comment]
This issue is fixed in master >= v2.0.0

**Comment by sypets** [comment]
Please also see new issue #64 (menu). 

As for this issue, I tested locally with v2.1.0 and it works fine.
